wrangler-core/src/main/java/io/cdap/directives/validation/ValidateStandard.java

```diff
@@ -0,0 +1,213 @@
+
+  static final String SCHEMAS_RESOURCE_PATH = "schemas/";
+  static final String MANIFEST_PATH = SCHEMAS_RESOURCE_PATH + "manifest.json";
+
+  static final Map<String, Conformer.Factory<JsonObject>> FORMAT_TO_FACTORY = new HashMap<>();
+
+  private String column;
+  private String schema;
+
+  static {
+    FORMAT_TO_FACTORY.put(JsonConformer.SCHEMA_FORMAT, new JsonConformer.Factory());
+
+    try {
+      standardsManifest = getManifest();
+    } catch (IOException e) {
+      LOG.error("Unable to read standards manifest", e);
+    }
+  }
+
+  @Override
+  public UsageDefinition define() {
+    UsageDefinition.Builder builder = UsageDefinition.builder(NAME);
+    builder.define(COLUMN, TokenType.COLUMN_NAME);
+    builder.define(
+        STANDARD_SPEC,
+        TokenType.IDENTIFIER,
+        String.format(
+            "[one of: %s]", String.join(", ", standardsManifest.getStandards().keySet())));
+
+    return builder.build();
+  }
+
+  @Override
+  public void initialize(Arguments args) throws DirectiveParseException {
+    column = ((ColumnName) args.value(COLUMN)).value();
+    String spec = ((Identifier) args.value(STANDARD_SPEC)).value();
+
+    if (spec.equals("")) {
+      throw new DirectiveParseException("No standard specified to validate against");
+    }
+    if (standardsManifest == null) {
+      throw new DirectiveParseException(
+          "Standards manifest was not loaded. Please check logs for information");
+    }
+
+    Map<String, Standard> availableSpecs = standardsManifest.getStandards();
+    if (!availableSpecs.containsKey(spec)) {
+      throw new DirectiveParseException(
+          String.format(
+              "Unknown standard %s. Known values are %s",
+              spec, String.join(", ", standardsManifest.getStandards().keySet())));
+    }
+
+    Standard standard = availableSpecs.get(spec);
+    schema =
+        Paths.get(SCHEMAS_RESOURCE_PATH, String.format("%s.%s", spec, standard.getFormat()))
+            .toString();
+
+    if (!schemaToConformer.containsKey(schema)) {
+      if (!FORMAT_TO_FACTORY.containsKey(standard.getFormat())) {
+        throw new DirectiveParseException(
+            String.format("No validator for format %s", standard.getFormat()));
+      }
+
+      InputStream schemaStream;
+      try {
+        schemaStream = readResource(schema);
+      } catch (IOException e) {
+        throw new DirectiveParseException(
+            String.format("Unable to read standard schema: %s", e.getMessage()), e);
+      }
+
+      Conformer<JsonObject> conformer =
+          FORMAT_TO_FACTORY.get(standard.getFormat()).setSchemaStream(schemaStream).build();
+      conformer.initialize();
+      schemaToConformer.put(schema, conformer);
+    }
+  }
+
+  @Override
+  public List<Row> execute(List<Row> rows, ExecutorContext context)
+      throws DirectiveExecutionException, ErrorRowException, ReportErrorAndProceed {
+    for (Row row : rows) {
+      int idx = row.find(column);
+      if (idx != -1) {
```

Comment:
nit: it's more readable to do:

```
if (idx < 0) {
  continue;
}

...
```

as it removes one layer of nesting

Reply:
Done.